spec/unit/controllers/runtime/spaces_controller_spec.rb

```diff
@@ -1269,120 +1270,23 @@ def decoded_guids
       let(:isolation_segment_model) { IsolationSegmentModel.make }
       let(:organization) { Organization.make }
       let(:space) { Space.make(organization: organization) }
+      let(:assigner) { IsolationSegmentAssign.new }
 
       context 'associating an isolation_segment' do
-        context 'when the space contains no apps' do
```

Comment:
@ScarletTanager We were able to find the tests for when spaces contains apps, but we were unable to find the tests for spaces that had no apps in them, particularly testing associating the space with the isolation segment as different roles.  Could you point us to where those tests landed?